### PR TITLE
Fixes Hakiri reported issues

### PR DESCRIPTION
**Pull request by tauil**
This updates jquery-rails to 4.0.5 and locks version and fixes some security issues

### CODE REVIEW

**Comment by tauil** [comment]
I have no idea how to solve `Unescaped model attribute` issue in Hakiri report. Any guess @lucianosousa , @viniciussbs or @aoqfonseca ?

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -93,4 +93,8 @@ def destroy
   def event_params
     params.require(:event).permit(:name, :description, :occurs_at, :prod_description, :twitter, :url, :picture, :picture_cache)
   end
+
+  def load_user_event!(slug)
+    current_user.events.where(slug: slug).first!
```
Comment (lucianosousa) [inline app/controllers/events_controller.rb]: my fault here. it's better find_by(slug: sug) than this where().first
it returns the first and faster, and it's more elegant.

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -29,10 +29,9 @@
   <% if @event.proposals.blank? %>
     <span class="yet">This event is ready to receive proposals!</span><br/><br/>
     Tweet this event to your friends so they can submit proposals and spread the word!</spam>
-  <br/><br/>
-  <%= render 'shared/tweet_button',
-    text: "#{@event.name} is on a call for papers run! Check it out.",
-    url: event_url(@event) %>
+    <br/><br/>
+    <% event_text = "#{@event.name} is on a call for papers! Check it out." %>
```
Comment (lucianosousa) [inline app/views/events/show.html.erb]: :no_entry_sign:

Comment (tauil) [inline app/views/events/show.html.erb]: What's the problem? Do you really prefer the broken line sintax?

Comment (lucianosousa) [inline app/views/events/show.html.erb]: Do not instantiate variables in views, plz. :+1:

Comment (tauil) [inline app/views/events/show.html.erb]: :man_with_turban: a.k.a xiita. Ok. Later I'll improve this.

**Comment by tauil** [comment]
All Done. Review

File: app/views/events/_event.html.erb [hunk 3]
```diff
@@ -28,9 +28,7 @@
 
   <div class="event-social">
     <div class="twit">
-      <%= render 'shared/tweet_button',
-        text: "#{@event.name} is on a call for papers! Check it out.",
-        url: event_url(@event) %>
+      <%= render 'shared/tweet_button', text: t('events.share', event_name: @event.name), url: event_url(@event) %>
```
Comment (rodrigopinto) [inline app/views/events/_event.html.erb]: Não seria melhor passar para a partial uma variável `event` usando o `locals`?

**Comment by rodrigopinto** [comment]
O PR tem mais conteúdo do que o título explica, seria legal quebrar em dois PR's separados.

File: app/views/events/_event.html.erb (hunk 3 above)
Comment (tauil) [inline app/views/events/_event.html.erb]: I prefer this way. What you guys think?

**Comment by tauil** [comment]
The idea is to fix Hakiri warnings. The title is not clear enough, I'll improve it.

**Comment by tauil** [comment]
All done!

**Comment by lucianosousa** [comment]
Nice. I'll merge this today!

**Comment by lucianosousa** [comment]
Thanks!

**Pull request merged**